diesel/src/connection/sqlite/sqlite_value.rs

```diff
@@ -0,0 +1,92 @@
+extern crate libsqlite3_sys as ffi;
+extern crate libc;
+
+use std::ffi::CStr;
+use std::{slice, str};
+
+use backend::Sqlite;
+use row::Row;
+
+pub struct SqliteValue {
+    inner_statement: *mut ffi::sqlite3_stmt,
+    col_index: libc::c_int,
```

Comment:
Yes. It ended up being entirely internal, so it made sense to just have this be a `c_int` on the struct rather than a `usize`, and skip the coercions later.